registry/eip155/tron-evm.json

```
@@ -0,0 +1,39 @@
+{
+  "id": "tron-evm",
+  "shortName": "TRON EVM",
+  "fullName": "TRON EVM Mainnet",
+  "aliases": ["tronevm", "tron-evm-mainnet", "evm-728126428"],
+  "caip2Id": "eip155:728126428",
+  "graphNode": { "protocol": "ethereum" },
+  "explorerUrls": ["https://tronscan.org"],
+  "rpcUrls": [
+    "https://api.trongrid.io/jsonrpc",
+    "https://tron.drpc.org",
+    "https://rpc.ankr.com/tron_jsonrpc"
+  ],
+  "relations": [{ "kind": "evmOf", "network": "tron" }],
+  "apiUrls": [
+    { "url": "https://apilist.tronscanapi.com/api", "kind": "etherscan" }
```

```suggestion
  { "url": "https://apilist.tronscanapi.com/api", "kind": "other" }
```
Doesn't support etherscan api, i.e.:
https://apilist.tronscanapi.com/api?module=contract&action=getsourcecode&address=0x0000000000001267532f4387C34a5AA50A8D4284